index.html

```diff
@@ -229,6 +229,131 @@ <h2>Retrieval of Media Feeds</h2>
       site then it should remove the feed from the <a>media feed store</a>.
     </p>
 
+    <p>
+      After fetching and the user agent has a valid <a>media feed document</a>,
+      the user agent MUST run the following steps:
+
+      <ul>
+        <li>
+          For each <a>associated origin</a>:
+
+          <ul>
+            <li>
+              Perform a <a data-cite="FETCH#fetching">fetch</a> for the
+              <a><code>media-feeds</code> well-known URI</a> with the following
+              <a data-cite="FETCH#concept-request">request</a>
+              parameters:
+
+              <ul>
+                <li>
+                  <a data-cite="FETCH#concept-header-list">header list</a>
+                  containing the <a data-cite="rfc7231#header.accept">accept header</a>
+                  set to <code>application/json</code>
+                </li>
+                <li>
+                  <a data-cite="FETCH#concept-request-redirect-mode">redirect mode</a>
+                  set to <code>error</code>
+                </li>
+                <li>
+                  <a data-cite="FETCH#concept-request-credentials-mode">credentials mode</a>
+                  set to <code>omit</code>
+                </li>
+              </ul>
+            </li>
+            <li>
+              If the request failed, then return <var>false</var>
+            </li>
+            <li>
+              Parse the response body using a <a data-cite="JSON#">JSON</a> parser. If the
+              parsing fails, then return <var>false</var>
+            </li>
+            <li>
+              Get the <code>allowedAssociations</code> member. If it is not present, then
+              return <var>false</var>
+            </li>
+            <li>
+              Create a <var>allowed associations</var> set of origins
+            </li>
+            <li>
+              If <code>allowedAssociations</code> is a string, then parse it as a
+              <a data-cite="url#urls">URL</a>. If the parsing is successful, then add its
+              origin to <var>allowed associations</var>. If it is not successful, then
+              return <var>false</var> 
+            </li>
+            <li>
+              If <code>allowedAssociations</code> is an array, then for each member run
+              the following steps:
+
+              <ul>
+                <li>
+                  If the member is not a string, then skip it
+                </li>
+                <li>
+                  Parse the member as a <a data-cite="url#urls">URL</a>. If the parsing
+                  is successful, then add its origin to <var>allowed associations</var>
+                </li>
+              </ul>
+            </li>
+            <li>
+              Get the origin for the <a>media feed URL</a> and store it in <var>origin</var>
```

Comment:
Is it necessary to write out this step here? Couldn't we just say, "Return whether the origin for the media feed URL is in allowed associations"?